### PR TITLE
feat: list column get comparable value test

### PR DESCRIPTION
**Pull request by dotpointer**
Adding test for list column getComparableValue function since this is preventing a new release from being made.

### CODE REVIEW

**Comment by cristinecula** [comment]
@dotpointer can you fix the lint error?

**Comment by dotpointer** [comment]
> @dotpointer can you fix the lint error?

Fixed it now and pushed it.

File: test/list.test.js [hunk 1]
```diff
@@ -112,4 +112,10 @@ suite('horizontal', () => {
 		column.textProperty = null;
 		assert.equal(column.getString({ list: ['123', '345', '678']}), '123, 345, 678');
 	});
+
+	test('returns the value of an item', () => {
+		column.valueProperty = 'id';
+		assert.strictEqual(column.getComparableValue({ id: 13 }, 'id'), 13);
+		assert.isUndefined(column.getComparableValue({}, 'nonexisting-path'));
```
Comment (cristinecula) [inline test/list.test.js:119]: You should also add an assertion for the expected behaviour for this part of the code:

https://github.com/Neovici/cosmoz-omnitable/blob/07ef4946ff1f98437d56088cb4851f077d3a361e/cosmoz-omnitable-column-list.js#L159-L163

Comment (dotpointer) [reply test/list.test.js:119]: @cristinecula

I added a test:

`assert.strictEqual(column.getComparableValue({ items: [{ id: 13 }, { id: 14 }, { id: 15 }] }, 'items'), '13 14 15');`

But it fails, why?

Comment (dotpointer) [reply test/list.test.js:119]: I did a test - I renamed the `getComparableValue` function in `cosmoz-omnitable-column-list.js` and in `list.test.js` the result is that it cannot find the function in the test - so it ignores the list version of the function completely.

**Comment by megheaiulian** [comment]
No longer relevant.

**Pull request closed without merging**